### PR TITLE
 added shop list,filters of  books, view product, admin can add books…

### LINKED ISSUE #2: As a user, i want to see books that i can buy easily
- [ ] Create DB and link it to server side
- [ ] Create sign up component
- [ ] Create login component

### LINKED ISSUE #3: As a user i want to navigate the application and see different books by filters
- [ ] Create nav-bar component
- [ ] Create home page component 
- [ ] Create Card component for book

### LINKED ISSUE #10: Book Card component
The card will contain the next elements:

- Title
- body (book image)
- footer (Price and button to buy)

### LINKED ISSUE #30: add shop main page
add shop main page with all the books

### PR DESCRIPTION
**Pull request by mcorcias**
…, cards
- The user can see now shop page even if he didn't make a signin.
- The admin can add a new books to the store.
- The admin can add a new categories.
-The user can search a book by category and name.
- The list of the book displayed as a cards list with image and details.
-Before buying, the user can see details on the book he interesting

closes #2 closes #3 closes #10 closes #30

### CODE REVIEW

File: bookstore-front/src/core/Shop.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                setError(data.error)
+            } else {
+                setCategories(data)
+            }
+        });
+    };
+
+
+    const loadFilteredResults = newFilters => {
+        // console.log(newFilters);
```
Comment (jamiecoe) [inline bookstore-front/src/core/Shop.js]: Remove unnecessary `console.log` from production code

File: bookstore-front/src/core/Search.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                <div className="btn input-group-append" style={{ border: 'none' }}>
+                    <button className="input-group-text">Search</button>
+                </div>
+            </span>
+        </form>
+    );
+
+
+
+
```
Comment (jamiecoe) [inline bookstore-front/src/core/Search.js]: Remove unnecessary empty space

File: bookstore-front/src/core/Shop.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                setSkip(0);
+
+            }
+        });
+    };
+
+
+    const loadMore = () => {
+        const toSkip = skip + limit;
+        // console.log(newFilters);
```
Comment (jamiecoe) [inline bookstore-front/src/core/Shop.js]: remove unnecessary `console.log` from production code

File: bookstore-front/src/core/Shop.js (hunk 1 above)
Comment (ArielLB) [reply bookstore-front/src/core/Shop.js]: done

File: bookstore-front/src/core/Search.js (hunk 2 above)
Comment (ArielLB) [reply bookstore-front/src/core/Search.js]: ok

File: bookstore-front/src/core/Shop.js (hunk 3 above)
Comment (ArielLB) [reply bookstore-front/src/core/Shop.js]: ok

**Pull request merged**